content/django-vs-flask.md

```diff
@@ -0,0 +1,67 @@
+# Django or Flask which one is should you choose your project?
+Django and Flask are among the most popular Python web frameworks.
```

Comment:
I would add the links to Django and Flask pages here

Reply:
Yup, and maybe an image with the logos before every section

Reply:
Add this and I think is ready